packages/backend/src/plugins/azure-sites.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createRouter,
+  AzureSitesApi,
+} from '@backstage/plugin-azure-sites-backend';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
+import { CatalogClient } from '@backstage/catalog-client';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  const catalogApi = new CatalogClient({ discoveryApi: env.discovery });
+  return await createRouter({
+    logger: env.logger,
+    permissions: env.permissions,
+    azureSitesApi: AzureSitesApi.fromConfig(env.config),
+    catalogApi,
```

Comment:
I think in the changeset you want to mention that the `catalogApi` is required in this `createRouter` method as it wasn't before. The permissions API is not required therefore, not breaking.

Reply:
Changeset updated!